src/java/org/jivesoftware/openfire/plugin/rest/controller/UserServiceLegacyController.java

```diff
@@ -105,10 +106,16 @@ public void createUser(String username, String password, String name, String ema
                     group = GroupManager.getInstance().getGroup(groupName);
                 } catch (GroupNotFoundException e) {
                     // Create this group ;
+                    try {
                     group = GroupManager.getInstance().createGroup(groupName);
                     group.getProperties().put("sharedRoster.showInRoster", "nobody");
                     group.getProperties().put("sharedRoster.displayName", groupName);
                     group.getProperties().put("sharedRoster.groupList", "");
+                    } catch (GroupAlreadyExistsException e1) {
+						e1.printStackTrace();
```

Comment:
This should be logged with an SLF4j logger.

Reply:
Actually, this should have been rethrown instead of logged so that the invoker knows that the request has failed. I've already pushed this change to your branch.